### ISSUE TITLE
Unable to sign in with Google (Sign in with Google has been temporarily disabled for this app)

### ISSUE THREAD

**Issue by m4gnify** [jonathanTIE/googledrive-copy-downloader#2]
Hi

Just decided to try out the app.
Having a problem though; unable to sign in. 

Opening the program for the first time (I guess) requires logging in, opening a Google sign in page.
However, after clicking my account, it won't authorize. It says Google hasn't verified the app yet.

![firefox_4kwHp5BqOd](https://user-images.githubusercontent.com/8404962/71828434-a1773200-30a2-11ea-9ee1-40280af4039c.png)

It's in Swedish (since I'm Swedish..)

Basically it says in English: 

> Sign in with Google has been temporarily disabled for this app
> Google has not yet verified this Google Sign-In app.

Looking forward to trying it out later if "we" can get it working. 
Thanks!

**Comment by jonathanTIE** [comment]
Hi,
it's a normal issue because the app is unverified and there are more than 100 users which is the limit per application by Google. The workaround here :
*If you have the "This app has not been verified yet" error :**

* Now you need to go here : https://developers.google.com/drive/api/v3/quickstart/python

* click on enable drive API

* download client configuration

* rename this file client_secrets.json

* replace the one in the folder of the script (in the same place as the .exe)

I will try to see if I can automate a little this process.

**Comment by m4gnify** [comment]
Thanks, I think it works now, at least logging in did, haven't tried downloading anything yet.

Maybe just add this info to the "front page" of the github, the readme?

**Comment by jonathanTIE** [comment]
Just did that, thanks for the reminder and hope you enjoy the script ;)
